### ISSUE TITLE
Outdated(?) documentation

### ISSUE THREAD

**Issue by zilti** [lancaster-university/microbit-dal#359]
This paragraph from the "Installation on Linux" section is wrong/outdated:

> A yotta target contains the information required by yotta in order to build a project for a specific combination of hardware. This includes the type of compiler. The microbit projects can build with both armcc and gcc, **but as it gets installed with the yotta installer**, we'll use gcc by default and choose a micro:bit specific target that knows about the hardware on the board.

**Comment by carlosperate** [comment]
Thanks @zilti, while yotta doesn't automatically install GCC, the instructions do include a step to install gcc-arm-none-eabi and most user will have probably followed it: http://docs.yottabuild.org/#linux-cross-compile

Perhaps we can rephrase it to make this more clear, do you have any suggestions?
